docs/setup.md

```diff
@@ -882,3 +883,59 @@ reverse proxy. Please refer to the [hosting and security](/configuration#hosting
 Also read
 [this](https://channels.readthedocs.io/en/stable/deploying.html#nginx-supervisor-ubuntu),
 towards the end of the section.
+
+### Install on Protainer {#portainer}
```

Comment:
typo!

Reply:
There are a few typos here :) but there’s a larger question about this. There are actually 3 PRs for the same question, would be great to have your thoughts! See https://github.com/paperless-ngx/paperless-ngx/pull/2663 for the latest discussion of it